communication.md

```diff
@@ -140,6 +140,14 @@ facilities to gracefully handle these failure cases or allow the consumer to han
 as they see fit.  The SDKs must also provide functionality to make a request with no
 response handling or retrying.
 
+#### Timeouts
+
+Telemetry SDKs should use reasonable timeout limits when sending requests to New Relic.
+The recommended value for this timeout is **10 seconds**.  Low timeout values (e.g. 
+less than one second) can result in intermittent failures due to fluctuations in response times.
```

Comment:
I chose to put this in the "Graceful Degradation" section, and so added a new header for "Retries and backoff" to help keep things organized.  Happy to put it elsewhere in this doc if that makes more sense.